test/UnitTests/TestFramework.UnitTests/Assertions/CollectionAssertTests.cs

```diff
@@ -502,12 +543,16 @@ private static List<object> GenerateDeeplyNestedCollection(int depth)
 
     private ICollection? GetCollection() => new[] { "item" };
 
+    private ICollection? GetSubsetCollection() => new[] { "iem", "a", "b" };
+
     private object? GetMatchingElement() => "item";
 
     private object? GetNotMatchingElement() => "not found";
 
     private ICollection? GetMatchingSuperSet() => new[] { "item", "item2" };
 
+    private ICollection? GetSuperSetCollection() => new[] { "item", "item2", "c", "d" };
```

Comment:
the typo was already there, but I think it should be changed to Superset everywhere, not SuperSet.